intel-retail/automated-self-checkout
# feat: create gst capi results log files and integration with docker launch

## PR Checklist



## What are you changing?
create gst capi results log files

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

Run below command to start face_detection pipeline before this PR, you will not find results/ logs files, with this PR, you run same command now, you will see gst-capi_0.log and pipeline0.log files produced under results/ folder.

`PIPELINE_PROFILE="face_detection" RENDER_MODE=1 sudo -E ./run.sh --workload ovms --inputsrc rtsp://127.0.0.1:8554/camera_0 --platform core`



## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
